style/style.css

```diff
@@ -0,0 +1,160 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
+  margin-top: -40px;
+  margin-left: 0;
+  margin-right: 0;
+  cursor: pointer;
+  width: 100%;
+  justify-content: center;
+}
+
+.nav-bar {
+  position: relative;
+  display: flex;
+  flex: 1 1 11.11%;
+  top: 130px;
+  width: 990px;
+  height: 40px;
+  border-radius: 3px;
+  box-shadow: rgba(56, 56, 56, 1) 0 1px 2px;
+  background: linear-gradient(to top, rgb(120, 120, 120) 17px, rgb(100, 100, 100) 17px);
+  align-items: stretch;
+  z-index: 1;
+}
+
+.nav-bar > .btn-item {
+  width: 11.12%;
+  height: 35px;
+  margin: 0 -1px;
+  background-color: transparent;
+  border: none;
+  border-right: 1px solid rgb(66, 66, 66);
+  color: white;
+  font-weight: 400;
+}
+
+.nav-bar > .btn-item:hover {
+  background: linear-gradient(#000, #444, #444, #444, #000);
+}
+
+a {
+  text-decoration: none;
+  color: #08c;
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  width: 10%;
+  height: 15px;
+  margin: auto 10px;
+  border-radius: 15px;
+  padding: 3px;
+  color: #fff;
+  font-size: 13px;
+  box-shadow: 0 2px 1px rgba(0, 0, 0, 0.6) inset, rgba(255, 255, 255, 0.2) 0 1px 0;
+}
+
+.search > input {
+  width: 70%;
+  margin-left: 3px;
+  border: none;
+  background-color: transparent;
+}
+
+.main-section {
+  background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 50% 100% no-repeat;
+  background-color: transparent;
+  background-size: 1350px 1315px;
```

Comment:
- [ ] The CSS background should `cover` the whole page irrespective of the screen size. Screenshot from requirements:

![image](https://user-images.githubusercontent.com/34813339/88228520-94511780-cc6f-11ea-8eb3-03fc37b541b3.png)

It looks great on 1366px screens but on larger screens it doesn't expand, try making this a dynamic value instead of a fixed size. You can use the responsive view option on chrome dev tools and change the window resolution to test this.